doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>Instead of adding this here afterwards, I would add a link with intersphinx in the initial explanation, something like "we will compute the rolling mean with <code> {meth}xarray.DataArray.rolling</code>...".  More on intersphinx  within ArviZ at https://arviz-devs.github.io/arviz/contributing/developer_guide.html#hyperlinks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>